orchestra/utils/task_lifecycle.py

```diff
@@ -310,14 +312,23 @@ def complete_and_skip_task(task_id):
     Returns:
         task (orchestra.models.Task):
             The completed and skipped task.
+
     """
     task = Task.objects.get(id=task_id)
-    task.status = Task.Status.COMPLETE
-    task.save()
-    for assignment in task.assignments.all():
-        assignment.status = TaskAssignment.Status.SUBMITTED
-        assignment.save()
-    create_subsequent_tasks(task.project)
+    assignment = current_assignment(task)
+    if assignment and assignment.worker:
```

Comment:
for tasks that are actively being worked on, this new branch leaves their assignments and iterations in a slightly cleaner state, and leaves us a note on the task data that we manually submitted the task on behalf of the Worker